modules/develop/pages/manage-topics/cloud-topics.adoc

```diff
@@ -0,0 +1,92 @@
+= Cloud Topics
+:description: Cloud Topics are Redpanda topics that enable users to trade off latency for lower costs.
+:page-beta: true
+// tag::single-source[]
+
+ifndef::env-cloud[]
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+endif::[]
+
+Starting in v25.3, Redpanda provides Cloud Topics to support multi-modal streaming workloads in the most cost-effective way possible: as a configurable option on a single cluster. While standard Redpanda xref:config-topics.adoc[topics] that use local storage or Tiered Storage are ideal for latency-sensitive, low-throughput workloads (for example, for audit logs), Cloud Topics are optimized for latency-tolerant, high-throughput workloads where cloud provider infrastructure costs (especially cross-AZ networking charges) are a major consideration that can become the dominant cost driver at high throughput. These workloads can include observability streams, offline analytics, AI/ML model training data feeds, or development environments where streaming latency isn't critical.
+
+Instead of replicating every byte across expensive network links, Cloud Topics leverage durable, inexpensive Cloud object storage (S3, ADLS,GCS, MinIO) as the primary mechanism for backing up data. This eliminates over 90% of the cost of replicating data over network links in multi-AZ or multi-region clusters. The difference in end-to-end latency is typically between 500ms-1s, which is often acceptable for many streaming workloads, and can unlock new streaming use cases that were previously not cost effective.
+
+== Prerequisites
+
+- xref:get-started:rpk-install.adoc[] v25.3 or later.
+ifndef::env-cloud[]
+- xref:manage:tiered-storage.adoc#set-up-tiered-storage[Set up] and xref:manage:tiered-storage.adoc#enable-tiered-storage[enable Tiered Storage] on your Redpanda cluster.
```

Comment:
I'd remove the second link, just because I think too many links are distracting. 
```suggestion
- xref:manage:tiered-storage.adoc#set-up-tiered-storage[Enable Tiered Storage] on your Redpanda cluster.
```